### PR TITLE
fix: remove some unused colorsDeprecated

### PR DESCRIPTION
**Pull request by chambo-e**
## Type

- Enhancement

### CODE REVIEW

File: src/components/PieChart/Legends.tsx [hunk 1]
```diff
@@ -57,14 +56,13 @@ const Bullet = styled.div<{
   width: 10px;
   height: 10px;
   margin: 0 8px;
-  background: ${({ color, theme }) =>
-    theme.colorsDeprecated[color as Color] ?? color};
+  background: ${({ color }) => color};
```
Comment (chambo-e) [inline src/components/PieChart/Legends.tsx:59]: These are always specified, we never use colors names

File: src/components/ProgressionButton/index.tsx [hunk 2]
```diff
@@ -45,12 +44,10 @@ const ProgressionContainer = styled(Box)`
 interface ProgressionProps {
   duration: number
   delay: number
-  color: string
 }
 
 const Progression = styled.div<ProgressionProps>`
-  background-color: ${({ theme, color }) =>
-    theme.colorsDeprecated[color as Color] ?? color};
+  background-color: ${({ theme }) => theme.colors.success.backgroundStrong};
```
Comment (chambo-e) [inline src/components/ProgressionButton/index.tsx:50]: Always used in this color, so now it's this one only :)

File: src/components/ExtendedReminder/index.tsx [hunk 3]
```diff
@@ -64,9 +64,8 @@ const StyledBadge = styled(Badge)`
 
 const StyledButtonLink = styled(Button)`
   margin-top: auto;
-  color: ${({ theme: { colorsDeprecated } }) => colorsDeprecated.blue};
-  background-color: ${({ theme: { colorsDeprecated } }) =>
-    colorsDeprecated.transparent};
+  color: ${({ theme: { colors } }) => colors.info.backgroundStrong};
```
[the remark this answers is not in the document]
Comment (chambo-e) [reply src/components/ExtendedReminder/index.tsx]: @matthprost done :)

**Review by matthprost — APPROVED** [review]

**Review by philibea — APPROVED** [review]

**Comment by chambo-e** [comment]
@matthprost if you could have a pass on Loki update pls ? :)

**Comment by matthprost** [comment]
> @matthprost if you could have a pass on Loki update pls ? :)

I got your back

**Pull request merged**